### PR TITLE
gh-153 Try to use an Apache mirror before falling back on the archive

### LINKED ISSUE #153: Hadoop download takes 1 hour to complete
Downloading Hadoop takes around an hour to download at the moment. Can we make this faster to speed up the docker builds?

### PR DESCRIPTION
**Pull request by ctas582**
# Related Issue

- Resolve #153

### CODE REVIEW

File: docker/accumulo/Dockerfile [hunk 1]
```diff
@@ -47,13 +52,13 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 COPY ./files/ .
```
Comment (t92549) [inline docker/accumulo/Dockerfile:52]: The Accumulo Dockerfile allows users to supply their own builds of Accumulo, ZooKeeper and Hadoop. Would it be possible to also add this to the hdfs Dockerfile / `build-hadoop.sh`? This way if I supply my own builds, no downloads have to happen at all?
Would be greatly appreciated if this would work as I have had a pain getting anything downloading this week.

File: docker/accumulo/Dockerfile [hunk 2]
```diff
@@ -47,13 +52,13 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
...
 # Otherwise, download official distributions
 RUN if [ ! -f "./accumulo-${ACCUMULO_VERSION}-bin.tar.gz" ]; then \
-		curl -s -fLO ${ACCUMULO_DOWNLOAD_URL}; \
+		(wget -nv -O ./accumulo-${ACCUMULO_VERSION}-bin.tar.gz ${ACCUMULO_DOWNLOAD_URL} || wget -nv -O ./accumulo-${ACCUMULO_VERSION}-bin.tar.gz ${ACCUMULO_BACKUP_DOWNLOAD_URL}); \
```
Comment (t92549) [inline docker/accumulo/Dockerfile:55]: Just wondering why the change from `curl` to `wget`?

File: docker/accumulo/Dockerfile (hunk 1 above)
Comment (t92549) [reply docker/accumulo/Dockerfile:52]: Never mind, it seems that functionality exists already https://github.com/gchq/gaffer-docker/blob/e040f871024ef3d01abc05b9e6990949f25f401a/docker/hdfs/files/build-hadoop.sh#L25

**Review by t92549 — APPROVED** [review]

File: docker/accumulo/Dockerfile (hunk 2 above)
Comment (ctas582) [reply docker/accumulo/Dockerfile:55]: It was just so that we would have a consistent command across dockerfiles
